common/exception/src/exception.rs

```diff
@@ -149,6 +150,27 @@ build_exceptions! {
     NotFoundSession(44),
     CannotListenerPort(45),
 
+    // user api error codes
+    UnkownUser(300),
```

Comment:
typo: 
```suggestion
    UnknownUser(300),
```

Reply:
got it! thanks bro

Reply:
BTW, I am thinking of  renaming `xxxAction` to `xxxRequest`, and `xxxActionResult` to `xxxReply`

The "Result" part of `XxxActionResult` makes people thinking about `std::result::Result`